.chloggen/webhookevent-enable.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'bug_fix'
```

Comment:
I'd say it's `enhancement`. There were no bugs here

Reply:
Heard! Was just going off of previous PR's for wiring, sorry!

Curious if the connecting/wiring process is documented in any official capacity or detail? I'd be happy to help with this if not

Reply:
I't mentioned in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-new-components but not very detailed